src/addon/ESMX/README.md

```diff
@@ -165,7 +167,7 @@ This section contains a key for for each *component-name*, specifying component
 | `test_exe`       | executable used to run test case              | ESMX_TEST_EXE          |
 | `test_tasks`     | number of tasks used to run test case         | ESMX_TEST_TASKS        |
 
-**Caution:** Downloading components using the `git_repository` option will result in a detached head. Developers making changing to component code must create or checkout a branch before making code changes. Downloading component using git_repository fails if the source_dir already exists.
+**CAUTION:** Downloading components using the `git_repository` option will result in a detached head. Developers making changing to component code must create or checkout a branch before making code changes. Downloading component using git_repository fails if the source_dir already exists.
```

Comment:
Oh, I think I fixed this actually by using explicit calls to git within the ESMX Driver CMakeLists.txt code.

Reply:
I'd say, lets merge this in still as is, but then you remove that paragraph under `feature/esmx_cmake` afterward.